app/main.py

```diff
@@ -1 +1,15 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    masks_to_buy = 0
+    for friend in friends:
```

Comment:
I suggest to separate the loop from two sides with an empty line for better readability